index.html

```diff
@@ -31,14 +31,14 @@ <h2>Find bugs in Java Programs</h2>
 
 <p>SpotBugs is a fork of <a href="http://findbugs.sourceforge.net/">FindBugs</a> (which is now an abandoned project), carrying on from the point where it left off with support of its community. Please check <a href="http://spotbugs.readthedocs.io/en/latest/">the official manual</a> for details.</p>
 
-<p>SpotBugs requires JRE (or JDK) 1.8.0 or later to run.  However, it can analyze programs compiled for any version of Java, from 1.0 to 1.9.</p>
+<p>SpotBugs requires JRE (or JDK) 1.8.0 or later to run.  However, it can analyze programs compiled for any version of Java, from 1.0 to 21.</p>
```

Comment:
`, from 1.0 to 21` - may be we can skip that? Whoever needs exact info can look into the changelog/readme/docs?

Reply:
Okay, I'll remove this part.